lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -314,6 +320,7 @@ export class UserAgentApplication {
                 this.authResponseCallback(authErr, response);
```

Comment:
Can you add log statements for each of these conditions?

Reply:
Added. Let me know if you need it to say something else.